packages/admin-ui-extensions/src/components/Button/Button.ts

```diff
@@ -22,6 +22,9 @@ export type ButtonProps = RequiredTitleOrIcon & {
    */
   primary?: boolean;
 
+  /** Allows the button to submit a form */
+  submit?: boolean;
```

Comment:
Sorry, I totally missed this PR. How will this prop actually work? Is it supposed to find the parent Form's `onSubmit` and trigger it? I'm not sure how feasible this would be on mobile. cc @james-woo @gutivg

Reply:
It is not feasible for the button to find the parent form.

Reply:
@alanthai can we remove this prop then? I don't think it's necessary especially since the form has its own `onSubmit`. If developers need to make a button also submit the form they can just call the same `onSubmit` handler for `onPress` instead of us automating this.

Reply:
@vividviolet @james-woo We don't automate this. This is an HTML feature and it's handled by the browser by default. What's the convention on form submit buttons on mobile?